apis/cache/v1beta1/cloudmemorystore_instance_types.go

```diff
@@ -125,6 +125,11 @@ type CloudMemorystoreInstanceParameters struct {
 	// +optional
 	// +immutable
 	ConnectMode *string `json:"connectMode,omitempty"`
+
+	// AuthEnabled: Optional. Indicates whether OSS Redis AUTH is enabled
+	// for the instance. If set to "true" AUTH is enabled on the instance.
+	// Default value is "false" meaning AUTH is disabled.
+	AuthEnabled *bool `json:"authEnabled,omitempty"`
```

Comment:
```suggestion
       // +optional
	AuthEnabled *bool `json:"authEnabled,omitempty"`
```
Also, is this immutable?

Reply:
If it can be changed at any point, let's just go ahead and make it _not immutable_ for now, which is currently the case as it looks like it was only updated to be immutable in the status, and it should be removed from status entirely.

Reply:
sounds good!